### PR TITLE
Yifan api mapbox direction 2

### PR DESCRIPTION
**Pull request by genieugod**
updated distance matrix and add waypoints for old routes

### CODE REVIEW

**Comment by rahimrasool** [comment]
Yet to be graded!

File: code/GetWayPoint.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    # Initialize the parser
+    config = configparser.ConfigParser()
+    # Read the config file
+    config.read("config.ini")
+    mapbox_token = config["mapbox"]["token"]
+
+    # Take file name from terminal
+    file_name = "../data/route1.csv"
+    df1 = pd.read_csv(file_name)
+    file_name = "../data/route2.csv"
```
Comment (rahimrasool) [inline code/GetWayPoint.py:69]: Don't hardcode the filenames here, even if you think the filenames will have consistent naming format

File: code/README.md [hunk 2]
```diff
@@ -15,3 +16,9 @@
   
   this file will using the data and request the time duration between each stop from mapbox direction API and than accumulates all 
   the time plus 5 min picking up or drop up time at each stop.
+  
+  # GetWayPoint.py
+  data: optimized route data
+  
+  this file will using the data and request the waypoints through the route along with the instructions for driveing
+
```
Comment (rahimrasool) [inline code/README.md:24]: Additionally, tell how you will run the file on the command line and what arguments are going to be added

File: code/GetWayPoint.py (hunk 1 above)
Comment (genieugod) [reply code/GetWayPoint.py:69]: updated this part as dir will be input from config file

**Pull request merged**